docs/deploy-cloudstate-cluster.adoc

```diff
@@ -0,0 +1,152 @@
+= Deploy Cloudstate Chat sample on Lightbend Cloudstate managed service
+Lightbend, 2020
+Version 2.0, 2020-07-06
+:description: Cloudstate Chat Application 
+:keywords: Cloudstate, stateful, serverless, chat-sample, tutorial, LBCS
+:sectnums:
+:toc:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+[deploy-cscluster-overview]
+== Overview
+
+You have built the 3 microservices composing the Chat application: 
+
+* `chat-ui` a stateless service serving the UI
+* `friends` a stateful service to manage the friend list of an user
+* `presence` a stateful service to maintain the online state of an user
+
+You have built the docker images of these services and published these images to your Docker registry. You have also run each of these images and tested that the service is in working order.
+
+If this is not the case, please review <<../README.adoc#cs-chat-sample-overview,Cloudstate Sample Chat Application>> to complete the prerequisites.
+
+We are now ready to deploy the Chat application, which we have chosen to deploy using Lightbend Cloudstate managed services (managed hosting). Compared to the alternative of deploying on your own Kubernetes cluster, Cloudstate managed services relieve the developer from the burden related to the infrastructure:
+
+* no need to provision hardware to setup (and maintain!) a Kubernetes cluster
+* simplify network configuration
+* simplify deployment
+
+
+== Prerequisites
+
+The main requirement is to have a Cloudstate account and some CLI tools to deploy the services on the Cloudstate platform.
```

Comment:
is there more than one tool to be used than `csctl`?

Reply:
I am undertermined on this one. The official line is that `csctl` is the only one tool. For the moment, I have some routing issues which I was told we could use `kubectl` to create a routes resource. But I am hesitant to mention in the documentation about the need of `kubectl`.

For now the deployment instructions on Cloudstate managed service. Should be considered as "beta". I would like to gather more feedback from Cloudstate specialists for the direction about the proper way to deploy on Cloudstate managed services.

Reply:
You could also link to the official Lightbend Cloudstate documentation here